rachiopy/rachioobject.py

```diff
@@ -100,3 +101,74 @@ def delete_request(self, path: str, body=None):
         :rtype: tuple
         """
         return self._request(path, "DELETE", body)
+
+    def _valve_request(self, path: str, method: str, body=None):
+        """Make a request from the API.
```

Comment:
```suggestion
        """Make a request to the API.
```

I see that the `_request` method also has this wrong. Could you improve that one as well?